### ISSUE TITLE
SteamVR plugin causes crash of the highest calibre

### ISSUE THREAD

**Issue by ChiefBreakeverything** [ValveSoftware/steamvr_unity_plugin#139]
- Installed SteamVR plugin 2.0.1 into Unity 2018.2.10f1
- No extra customization, just let SteamVR generate it's actions.json file and vrmanifest and so on
- Enter playmode
- Tracking seems to work fine as usual, although there's a lot of InvalidHandle errors and some null reference exceptions ([see issue here](https://github.com/ValveSoftware/steamvr_unity_plugin/issues/138))
- Exit playmode
- Enter playmode a second time
- Unity hangs (just Unity, everything else is fine)
- The SteamVR window says 'Ready (unresponsive) <my-project-name>'
- Unity can't be killed by task manager
- Open Unity Hub - can't open project because its already open
- Use [Process Hacker ](https://processhacker.sourceforge.io/) to attempt to destroy Unity and close the breach
- 'Unable to terminate Unity.exe: An attempt was made to access an exiting process'
- Only way to proceed is to hard-reboot the computer. A normal shut down or restart simply waits on "Rebooting" or "Shutting down" indefinitely (I left it for an hour, definitely not going to close by itself).

The error is highly reproducible. My guess is that something deep within SteamVR doesn't close down properly after exiting playmode for the first time, causing Unity to meet some sort of horrific end when trying to enter for a second time.

**Comment by superjayman** [comment]
Agreed!! WTF is going on with Valve? FIX THIS SHIT NOW, and at least respond to all the issues we're all experiencing. 
I have also had multiple crashes!! all sorts of weird errors. TEST BEFORE RELEASE!!!!!!! Same setup as above

**Comment by zite** [comment]
Strange, this was tested with 2018.2.7, I’ll try with .10 today.

**Comment by zite** [comment]
I gave it a go with .10 and it worked on my side. Can you create a system report and attach that so we can better diagnose the problem?

![systemreport](https://user-images.githubusercontent.com/198704/46378588-f3840900-c650-11e8-8ecd-153737451c49.png)
